### PR TITLE
14.0 mig l10n it fatturapa in purchase

### PR DESCRIPTION
**Pull request by fredzamoabg**
migrazione modulo l10n_it_fatturapa_in_purchase dalla version 12.0

--
Confermo di aver firmato il CLA https://odoo-community.org/page/cla e di aver letto le linee guida su https://odoo-community.org/page/contributing

### CODE REVIEW

File: l10n_it_fatturapa_in_purchase/views/invoice_view.xml [hunk 1]
```diff
@@ -3,13 +3,10 @@
...
-    <record id="view_invoice_line_form_fatturapa_po_line" model="ir.ui.view">
-        <field name="name">view_invoice_line_form_fatturapa_po_line</field>
-        <field name="model">account.invoice.line</field>
-        <field
-            name="inherit_id"
-            ref="l10n_it_fatturapa.view_invoice_line_form_fatturapa"
-        />
+    <record id="view_invoice_form_fatturapa_po_line" model="ir.ui.view">
+        <field name="name">view_invoice_form_fatturapa_po_line</field>
+        <field name="model">account.move</field>
```
Comment (SimoRubi) [inline l10n_it_fatturapa_in_purchase/views/invoice_view.xml:8]: Come mai hai rimosso la modifica dal form della riga fattura?

**Review by SimoRubi — CHANGES_REQUESTED** [review]

**Review by SimoRubi — APPROVED** [review]
Revisione del codice, per me è ok

**Review by TheMule71 — APPROVED** [review]
LGTM

**Comment by tafaRU** [comment]
@sbidoul merge failed due to https://github.com/OCA/l10n-italy/runs/4331912624?check_suite_focus=true#step:4:161 Could you please check it?

**Comment by sbidoul** [comment]
@tafaRU That is because the setup.py is missing for l10n_it_fatturapa_in_purchase. But what I don't understand is why the pre-commit check does not detect it.

**Comment by tafaRU** [comment]
@sbidoul, thanks for your help! Do you think it could be useful open a issue? On MQT?

**Comment by tafaRU** [comment]
@fredzamoabg riesegui per favore `pre-commit` in modo che venga aggiunto `setup/l10n_it_intrastat_statement`.

**Comment by sbidoul** [comment]
@tafaRU issue created https://github.com/OCA/oca-addons-repo-template/issues/93

**Pull request merged**